<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle"><string>Viewer (Designer)</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topBar">
      <item><widget class="QPushButton" name="btnOpen"><property name="text"><string>열기</string></property></widget></item>
      <item><widget class="QPushButton" name="btnClose"><property name="text"><string>닫기</string></property></widget></item>
      <item><widget class="QComboBox" name="comboStage">
        <item><property name="text"><string>Input</string></property></item>
        <item><property name="text"><string>Gray</string></property></item>
        <item><property name="text"><string>Binary</string></property></item>
        <item><property name="text"><string>Canny</string></property></item>
      </widget></item>
      <item><widget class="QSpinBox" name="spinThresh"><property name="maximum"><number>255</number></property><property name="value"><number>128</number></property></widget></item>
      <item><widget class="QSpinBox" name="spinC1"><property name="maximum"><number>255</number></property><property name="value"><number>80</number></property></widget></item>
      <item><widget class="QSpinBox" name="spinC2"><property name="maximum"><number>255</number></property><property name="value"><number>160</number></property></widget></item>
      <item><spacer name="spacer"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblView">
      <property name="text"><string/>
      </property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>